changedetectionio/tests/conftest.py

```diff
@@ -50,6 +52,18 @@ def app(request):
 
     app_config = {'datastore_path': datastore_path, 'disable_checkver' : True}
     cleanup(app_config['datastore_path'])
+
+    # By ARG or ENV in Dockerfile or test-only.yml
+    if os.getenv("LOGGER_LEVEL"):
+        logger_level = os.getenv("LOGGER_LEVEL")
+    else:
+        # A default logger level for pytest
+        # Just in case if it doesn't build with TRACE log ARG in test-only.
+        logger_level = 'TRACE'
```

Comment:
I'm not sure about this. Should I remove the `os.getenv` part in this file?

Reply:
yeah, conftest.py should always be in `TRACE` maximum mode